config/software/ruby.rb

```diff
@@ -138,6 +146,8 @@
       patch source: "ruby-3.0.1-configure.patch", plevel: 1, env: patch_env
     when "3.0.5"
       patch source: "ruby-3.0.5-configure.patch", plevel: 1, env: patch_env
+    when "3.0.6"
+      patch source: "ruby-3.0.5-configure.patch", plevel: 1, env: patch_env
```

Comment:
@rfpronk just noticing that this is sourcing Ruby 3.0.5?

Reply:
@rfpronk  Lets retain the default version to 3.1.x  and keep the  ruby 3.2.2 in the version list.
yet we have'nt made a plan to pick 3.2.x .